cnxarchive/tests/test_sitemap.py

```diff
@@ -61,7 +104,9 @@ def test_updatefreq_urlEntry(self):
         """test valid changefreq"""
         ue = sitemap.UrlEntry(loc='http://example.com', changefreq='weekly')
         self.assertEqual(repr(ue), "<UrlEntry 'http://example.com'>")
-        self.assertEqual(str(ue), '<url>\n<loc>http://example.com</loc>\n<changefreq>weekly</changefreq>\n</url>\n')
+        self.assertEqual(
+            str(ue), '<url>\n  <loc>http://example.com</loc>\n'
+                     '  <changefreq>weekly</changefreq>\n</url>\n')
```

Comment:
Gee, it'd be nice to compute an actual likely value for this. TODO, I suppose.

Reply:
I actually thought it should be `never`, the urls in our sitemap.xml have versions in them right?

Reply:
What do you think about changing the `changefreq` to `never`?

Reply:
Oh then does it mean we should remove the versions from the urls?

Reply:
I think we need an actual SEO expert to think about these things. :/